ovn-org/ovn-kubernetes
# Provide a way to skip setting port security on Logical Switch Port

There are Pods which requires the spoof check to be disabled, this commit allows users to annotate pod to skip port-security for a particular network, for example:

k8s.ovn.org/skip-spoofchk-on-networks=<namespace/NADName>[,...]

Note: specify `default` in the list if port security need to be skipped on default network.

#### What this PR does and why is it needed
For some scenarios, port security need to be removed from logical switch port.

#### How to verify it
1. add following annotation to the pod:
 k8s.ovn.org/skip-spoofchk-on-networks=default
2. make sure port-security field of logical_switch_port table is blank


#### Description for the changelog
k8s.ovn.org/skip-spoofchk-on-networks=

#### Does this PR introduce a user-facing change?
NONE
